lib/wallaroo/core/sink/connector_sink/connector_sink.pony

```diff
@@ -484,6 +513,13 @@ actor ConnectorSink is Sink
         barrier_token.string().cstring(), _sink_id.string().cstring(),
         input_id.string().cstring(), _phase.name().cstring())
     end
+    match _phase
```

Comment:
We shouldn't be matching on phases, but delegating to them.

Reply:
I'll have a chicken-and-the-egg problem wrt `InitialSinkPhase` will need a `Sink` argument in its constructor ... so I'll introduce a new phase, `EarlySinkPhase` to work around it.